docs/portal/image-registry/deleting-projects.md

```diff
@@ -0,0 +1,5 @@
+# Deleting Projects
+
```

Comment:
The contents below are a bit inconsistent with what's documented previously. Would be better if we could keep the consistency across different pages. 
For example, if you compare this page with the page "deleting-an-image-pull-secret", you'll find inconsistency.

Reply:
I made this topic and the Deleting Users topic consistent with the other topics that deal with deletions.